### PR TITLE
Addition of HybridDataManager SDK

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.reference.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Management.HybridData</PackageId>
+    <Description></Description>
+    <Version>0.9.0</Version>
```
Comment (dsgouda) [inline src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj]: Please set this to `0.9.0-preview`

File: src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <AssemblyName>Management.HybridData</AssemblyName>
+    <PackageTags>MicrosoftAzure Management</PackageTags>
```
Comment (dsgouda) [inline src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj]: Please use [this](https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj#L11) as an example

File: src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.reference.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Management.HybridData</PackageId>
```
Comment (dsgouda) [inline src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj]: For consistency please name this as `Microsoft.Azure.Management.HybridData`

File: src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    <Description></Description>
+    <Version>0.9.0</Version>
+    <AssemblyName>Management.HybridData</AssemblyName>
+    <PackageTags>MicrosoftAzure Management</PackageTags>
+    <PackageReleaseNotes>
+      <![CDATA[
```
Comment (dsgouda) [inline src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj]: Please add the `PackageReleaseNotes`

File: src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      
+      ]]>
+    </PackageReleaseNotes>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net452|AnyCPU'">
```
Comment (dsgouda) [inline src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj]: Please remove this `PropertyGroup`

File: src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    <AssemblyName>Management.HybridData</AssemblyName>
+    <PackageTags>MicrosoftAzure Management</PackageTags>
+    <PackageReleaseNotes>
+      <![CDATA[
+      
+      ]]>
+    </PackageReleaseNotes>
+  </PropertyGroup>
+
+  <PropertyGroup>
```
Comment (dsgouda) [inline src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj]: Please use [this](https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj#L18) as a sample

File: src/SDKs/HybridDataManager/Management.HybridData/generate.ps1 [hunk 7]
```diff
@@ -0,0 +1 @@
+Start-AutoRestCodeGeneration -ResourceProvider "hybriddatamanager/resource-manager" -AutoRestVersion "latest" -SpecsRepoFork "AzureHybridDataManager"
```
Comment (dsgouda) [inline src/SDKs/HybridDataManager/Management.HybridData/generate.ps1]: Is the Rest spec PR merged already?

File: README.md [hunk 8]
```diff
@@ -1,31 +1,27 @@
...
+| Management Libraries | [![Build Status](https://travis-ci.org/Azure/azure-sdk-for-net.svg?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-net) |
+| Client Libraries | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/41?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=41&branchName=master) |
 
-# Microsoft Azure SDK for .NET
- ----------
-The Microsoft Azure SDK for .NET allows you to build applications
-that take advantage of scalable cloud computing resources.
+Microsoft Azure is an ever-expanding set of cloud services to help your organization meet your business challenges. It’s the freedom to build, manage, and deploy applications on a massive, global network using your favorite tools and frameworks.
+
+This repo contains the libraries to allow you to easily leverage Azure from your applications and tools.
```
Comment (dsgouda) [inline README.md]: These changes are unexpected. Please revert this file to the remote master

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/HybridDataManager/Management.HybridData/generate.ps1 (hunk 7 above)
Comment (t-prta) [reply src/SDKs/HybridDataManager/Management.HybridData/generate.ps1]: No, we are waiting for a linter suppression approval for that PR

File: README.md (hunk 8 above)
Comment (t-prta) [reply README.md]: Pulling the master/readme.md and committing it.

Comment (t-prta) [reply README.md]: The file is an exact copy of azure/azure-sdk-for-net/master readme.md. I am not sure why the diff is showing up this way.

Comment (dsgouda) [reply README.md]: How are you reverting (git checkout ?)

**Comment by dsgouda** [comment]
Please link the rest spec pr here

File: README.md (hunk 8 above)
Comment (t-prta) [reply README.md]: git checkout upstream/master -- README.md
I ran this command, and it doesn't modify the file.

Comment (dsgouda) [reply README.md]: Can you post what your upstream is here

Comment (t-prta) [reply README.md]: Upstream is the Azure/azure-sdk-for-net repo

**Comment by dsgouda** [comment]
Closing since a new PR is opened

**Pull request closed without merging**